api.md

```diff
@@ -840,7 +842,127 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
+|`timescaledb.refresh_interval`|Refresh interval controls how often the background materializer is run. It is specified using an `INTERVAL` value. By default, this is set to twice the bucket width (if the datatype is an INTERVAL), otherwise it is set to 12 hours.|
+|`timescaledb.max_interval_per_job`|Max interval per job specifies the amount of data processed by the background materializer job when the continuous aggregate is updated. The default value is `20 * bucket width`.|
+
+#### Restrictions   
+- SELECT query should be of the form specified in the syntax above.
+- The hypertable used in the `SELECT` may not have [row-level-security policies][postgres-rls] enabled. 
+-  GROUP BY clause should include a time_bucket expression.  The time_bucket 
+[time-bucket] expression should use the time dimension column of the hypertable.
+- [`time_bucket_gapfill`][time-bucket-gapfill] is not allowed in continuous
+  aggs, but may be run in a `SELECT` from the continuous aggregate view. 
+- In general, aggregates which can be [parallelized by
+  PostgreSQL][postgres-parallel-agg] are allowed in the view definition, this
+  includes most aggregates distributed with PostgreSQL. Aggregates with ORDER BY,
+  DISTINCT and filter clauses are not permitted.  
+* All functions and their arguments included in `SELECT`, `GROUP BY` and `HAVING` clauses must be [immutable][postgres-immutable]. 
+- Queries with ORDER BY  are disallowed.
+- The view is not allowed to be a [security barrier view][postgres-security-barrier]. 
+[time-bucket]: /api#time_bucket
+[time-bucket-gapfill]: /api#time_bucket_gapfill
+[postgres-immutable]:https://www.postgresql.org/docs/current/xfunc-volatility.html
+[postgres-parallel-agg]:https://www.postgresql.org/docs/current/parallel-plans.html#PARALLEL-AGGREGATION
+[postgres-security-barrier]:https://www.postgresql.org/docs/current/rules-privileges.html
+ 
+## ALTER VIEW
+`ALTER VIEW` statement can be used to modify the WITH clause  
+[options](#create-view-with) for the continuous aggregate view.
+
+``` sql
+ALTER VIEW <view_name> SET ( option =  <value> )
+```
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
+
+Set the max interval processed by a materializer job (that updates the continuous aggregate) to 1 week.
+```sql
+ALTER MATERIALIZED VIEW contagg_view SET (timescaledb.max_interval_per_job = '1 week');
+```
+Set the refresh lag to 1 hour, the refresh interval to 30 minutes and the max 
+interval processed by a job to 1 week for the continuous aggregate.
+```sql
+ALTER VIEW contagg_view SET (timescaledb.refresh_lag = '1h', timescaledb.max_interval_per_job='1week', timescaledb.refresh_interval='30m');
+
+```
+
+## REFRESH VIEW
```

Comment:
REFRESH MATERIALIZED VIEW?

Reply:
Didn't want to bring attention to the odd "MATERIALIZED" in the header. If we are renaming the section headers how about REFRESH for continuous aggregates here? Do they need to match the actual command? @cevain @solugebefola

Reply:
fixed.